src/default_panels/StyleTracesPanel.js

```diff
@@ -43,12 +44,15 @@ import {
 const StyleTracesPanel = (props, {localize: _}) => (
   <TraceAccordion canGroup>
     <TextEditor label={_('Name')} attr="name" richTextOnly />
-    <ShowInLegend
-      label={_('Show in Legend')}
-      attr="showlegend"
-      options={[{label: _('Show'), value: true}, {label: _('Hide'), value: false}]}
-    />
     <NumericFraction label={_('Trace Opacity')} attr="opacity" />
+    <PlotlySection name={_('Legend')}>
+      <ShowInLegend
```

Comment:
is the groupcreator meant to be IN the `ShowInLegend`? If i set it to 'hide' I still see the groupcreator